robottelo/vm.py

```diff
@@ -333,6 +333,26 @@ def configure_puppet(self, rhel_repo=None):
         # stage can be ignored.
         self.run(u'puppet agent -t 2> /dev/null')
 
+    def execute_foreman_scap_client(self, policy_id=None):
+        """Executes foreman_scap_client on the vm/clients to create security
+        audit report.
+
+        :param policy_id: The Id of the OSCAP policy.
+        :return: None.
```

Comment:
I thinks it not required to mention that its returning None as its not really. You can choose to remove this.